### PR TITLE
Add Og::isGroupContent

### PR DESCRIPTION
**Pull request by amitaibu**

### CODE REVIEW

**Comment by amitaibu** [comment]
Ready for review

File: src/Og.php [hunk 1]
```diff
@@ -182,10 +184,32 @@ public static function isGroup($entity_type_id, $bundle_id) {
...
+   * @param string $entity_type_id
+   *   The entity type.
+   * @param string $bundle_id
+   *   The bundle name.
+   *
+   * @return bool
+   *   True or false if the given entity is group.
+   */
+  public static function isGroupContent($entity_type_id, $bundle_id) {
+    return (bool)static::getAllGroupAudienceFields($entity_type_id, $bundle_id);
```
Comment (damiankloip) [inline src/Og.php]: nit: space after (bool).

File: tests/src/Kernel/Entity/GroupAudienceTest.php [hunk 2]
```diff
@@ -77,6 +77,10 @@ public function testGetAllGroupAudienceFields() {
 
     $field_names = Og::getAllGroupAudienceFields('entity_test', $bundle);
     $this->assertEquals(array($field_name1, $field_name2), array_keys($field_names));
+
+    // Test Og::isGroupContent method, which is just a wrapper around
+    // Og::getAllGroupAudienceFields.
+    $this->assertTrue(Og::isGroupContent('entity_test', $bundle));
```
Comment (damiankloip) [inline tests/src/Kernel/Entity/GroupAudienceTest.php:83]: Could we add an assertFalse case in here somewhere too?

Comment (amitaibu) [inline tests/src/Kernel/Entity/GroupAudienceTest.php:83]: added. I knew I had too, I was just lazy. Thanks for being strict :wink:

**Comment by amitaibu** [comment]
I've addressed the comments.

File: tests/src/Kernel/Entity/GroupAudienceTest.php (hunk 2 above)
Comment (damiankloip) [inline tests/src/Kernel/Entity/GroupAudienceTest.php:83]: hahaha, you would do the same to me! :)

Comment (amitaibu) [inline tests/src/Kernel/Entity/GroupAudienceTest.php:83]: for sure! :)

**Pull request merged**